modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -477,10 +458,13 @@ export default class AgentManager {
 
                             const useBigIntJson = taskId.startsWith('query');
 
-                            return this.pool.postMessage({payload: res.data, useBigIntJson});
+                            const response = await this.pool.postMessage({payload: res.data, useBigIntJson});
+                            return response;
                         }
 
-                        return res;
+                        if (NON_ZIPPED_TASKS.has(params.taskId))
```

Comment:
This code is not needed: AgentManager.drivers, AgentManager.schemas, AgentManager.schemas.tables call AgentManager._sendToAgent directly

Reply:
Done.